src/qt/dvrenderers.cpp

```diff
@@ -54,6 +54,13 @@ bool wxDataViewBitmapRenderer::GetValue( wxVariant &value ) const
     return false;
 }
 
+bool
+wxDataViewBitmapRenderer::IsCompatibleVariantType(const wxString& variantType) const
+{
+    return variantType == wxS("wxBitmap")
```

Comment:
Does it needs `wxBitmapBundle` like the other platforms?

Reply:
I guess... But then this seems to be completely unimplemented in wxQt anyhow, and `wxBitmapBundle` is not implemented for it neither, so it's a bit of a moot point anyhow.